pkg/controller/issueapi/issue.go

```diff
@@ -254,6 +255,14 @@ func (c *Controller) HandleIssue() http.Handler {
 			return
 		}
 
+		err = c.recordCapacity(ctx, realm)
+		if err != nil {
+			logger.Errorw("failed to record realm capacity", "error", err)
+			stats.Record(ctx, c.metrics.CodeIssueErrors.M(1))
+			c.h.RenderJSON(w, http.StatusInternalServerError, api.Errorf("failed to update realm capacity"))
```

Comment:
Are you sure we want to fail if we don't update this stat?

Reply:
In my mind associated that error with the whole process, but now it's not fatal.